### PR TITLE
fix(frontend): login banner spacing

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

#3240 broke the spacing around the banner displayed in the login modal.

# Changes

- Redo div around banner and absolute image

# Notes

Note against better idea.

# Screenshots

Issue:

Capture d’écran 2024-11-20 à 18 08 19
Capture d’écran 2024-11-20 à 18 08 15
Capture d’écran 2024-11-20 à 18 08 11

Fix:

Capture d’écran 2024-11-20 à 18 06 26
Capture d’écran 2024-11-20 à 18 06 20
Capture d’écran 2024-11-20 à 18 06 17

### CODE REVIEW

File: src/frontend/src/lib/components/loaders/Loader.svelte [hunk 1]
```diff
@@ -129,7 +129,9 @@
 		<div in:fade={{ delay: 0, duration: 250 }}>
 			<Modal testId={LOADER_MODAL}>
 				<div class="stretch">
-					<ImgBanner src={banner} />
+					<div class="relative mb-8 block md:h-40">
+						<ImgBanner src={banner} styleClass="h-full md:absolute aspect-auto" />
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/loaders/Loader.svelte:133]: is `aspect-auto` needed? i thought it was by default

**Review by AntonioVentilii — APPROVED** [review]
Already approving, all good! Just a question above

File: src/frontend/src/lib/components/loaders/Loader.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/loaders/Loader.svelte:133]: Yes because `ImgBanner` now is per default set to an aspect ratio 16-9.

Comment (peterpeterparker) [reply src/frontend/src/lib/components/loaders/Loader.svelte:133]: If I don't do so it defeats the absolute positioning and spacing.

**Pull request merged**